src/components/organisms/calendarDisclosure/calendarDisclosure.scss

```
@@ -37,9 +37,25 @@
       padding-right: $x-small-space;
       background-color: transparent;
       color: $primary-dark-color;
+      border-radius: $regular-radius;
       border: none;
```

il faut mettre une border de base sinon ça fait décaler l'élément dans la page au hover
```suggestion
      border: 1px solif transparent;
```